src/daft-io/src/stats.rs

```diff
@@ -27,7 +27,7 @@ impl Drop for IOStatsContext {
         let num_puts = self.load_put_requests();
         let mean_get_size = (bytes_read as f64) / (num_gets as f64);
         let mean_put_size = (bytes_uploaded as f64) / (num_puts as f64);
-        log::debug!(
+        log::trace!(
```

Comment:
this was just a huge info dump so i changed it to trace as it was just too noisy.